buildSrc/src/main/java/org/apache/geode/gradle/testing/process/LauncherProxyWorkerProcessBuilder.java

```diff
@@ -0,0 +1,160 @@
+      ProcessLauncher processLauncher) {
+    this.delegate = delegate;
+    this.processLauncher = processLauncher;
+  }
+
+  @Override
+  public WorkerProcessBuilder applicationClasspath(Iterable<File> files) {
+    return delegate.applicationClasspath(files);
+  }
+
+  @Override
+  public Set<File> getApplicationClasspath() {
+    return delegate.getApplicationClasspath();
+  }
+
+  @Override
+  public WorkerProcessBuilder applicationModulePath(Iterable<File> files) {
+    return delegate.applicationModulePath(files);
+  }
+
+  @Override
+  public Set<File> getApplicationModulePath() {
+    return delegate.getApplicationModulePath();
+  }
+
+  @Override
+  public WorkerProcessBuilder setBaseName(String baseName) {
+    return delegate.setBaseName(baseName);
+  }
+
+  @Override
+  public String getBaseName() {
+    return delegate.getBaseName();
+  }
+
+  @Override
+  public WorkerProcessBuilder setLogLevel(LogLevel logLevel) {
+    return delegate.setLogLevel(logLevel);
+  }
+
+  @Override
+  public WorkerProcessBuilder sharedPackages(Iterable<String> packages) {
+    return delegate.sharedPackages(packages);
+  }
+
+  @Override
+  public Set<String> getSharedPackages() {
+    return delegate.getSharedPackages();
+  }
+
+  @Override
+  public JavaExecHandleBuilder getJavaCommand() {
+    return delegate.getJavaCommand();
+  }
+
+  @Override
+  public LogLevel getLogLevel() {
+    return delegate.getLogLevel();
+  }
+
+  @Override
+  public WorkerProcessBuilder sharedPackages(String... packages) {
+    return delegate.sharedPackages(packages);
+  }
+
+  @Override
+  public Action<? super WorkerProcessContext> getWorker() {
+    return delegate.getWorker();
+  }
+
+  @Override
+  public void setImplementationClasspath(List<URL> implementationClasspath) {
+    delegate.setImplementationClasspath(implementationClasspath);
+  }
+
+  @Override
+  public void setImplementationModulePath(List<URL> implementationModulePath) {
+    delegate.setImplementationModulePath(implementationModulePath);
+  }
+
+  @Override
+  public void enableJvmMemoryInfoPublishing(boolean shouldPublish) {
+    delegate.enableJvmMemoryInfoPublishing(shouldPublish);
+  }
+
+  /**
+   * Replaces the standard worker process's process launcher with this builder's launcher.
+   */
+  @Override
+  public WorkerProcess build() {
+    WorkerProcess workerProcess = delegate.build();
+    Object workerProcessDelegate = getFieldValue(workerProcess, "delegate");
+    Object execHandle = getFieldValue(workerProcessDelegate, "execHandle");
+    Class<?> processLauncherType = getField(execHandle, "processLauncher").getType();
+    setFieldValue(execHandle, "processLauncher", assignableProcessLauncher(processLauncherType));
+    return workerProcess;
+  }
+
+  /**
+   * Because the exec handle created by Gradle uses a classloader different from ours, we can't
+   * simply construct a Gradle {@code ProcessLauncher) to assign. Instead we create proxy, using the
```

Comment:
Syntax error after `{@code ProcessLauncher`. Only caught it because of GitHub's syntax highlighting.